src/command-line/index.js

```diff
@@ -16,7 +16,8 @@ var argv = program.parseOptions(process.argv);
 Helper.setHome(program.home);
 
 if (!fs.existsSync(Helper.CONFIG_PATH)) {
-	mkdirp.sync(Helper.HOME, {mode: "0700"});
+	fsextra.ensureDirSync(Helper.HOME);
+	fs.chmod(Helper.HOME, 0700)
```

Comment:
missing `;`, and `0700` should be quoted to avoid the other eslint error about octal literals.